modules/product/index.js

```diff
@@ -95,9 +98,14 @@ module.exports = {
   },
   handlers(self, options) {
     return {
-      afterSave: {
-        test(req, doc) {
-          self.apos.notify(req, 'New product added: ' + doc.title);
+      afterInsert: {
+        insertNotif(req, doc) {
+          self.apos.notify(req, 'New product added: ' + doc.title, { dismiss: 10 });
```

Comment:
Use `dismiss: true` to get the default number of seconds, use backticks for the string substitution